File: LondonDataServices.IDecide.Portal.Client/src/models/userAccess/userAccess.ts
export class UserAccess {
    public id: string = "";
    public displayName: string = "";
    public UserId: string = "";
    public entraUpn: string = "";
    public givenName: string = "";
    public surname: string = "";
    public email: string = "";
    public userPrincipalName: string = "";
    public orgCode: string = "";
    public jobTitle: string = "";
    public activeFrom?: Date | undefined;
    public activeTo?: Date | undefined;
    public createdBy?: string;
    public createdDate?: Date | undefined;
    public updatedBy?: string;
    public updatedDate?: Date | undefined;
}

File: LondonDataServices.IDecide.Portal.Tests.Integration/Models/UserAccesses/UserAccess.cs
﻿// ---------------------------------------------------------
// Copyright (c) North East London ICB. All rights reserved.
// ---------------------------------------------------------

using System;

namespace LondonDataServices.IDecide.Portal.Tests.Integration.Models.UserAccesses
{
    public class UserAccess
    {
        public Guid Id { get; set; }
        public string UserId { get; set; }
        public string GivenName { get; set; } = string.Empty;
        public string Surname { get; set; } = string.Empty;
        public string DisplayName { get; set; } = string.Empty;
        public string JobTitle { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string UserPrincipalName { get; set; } = string.Empty;
        public string OrgCode { get; set; } = string.Empty;
        public string CreatedBy { get; set; } = string.Empty;
        public DateTimeOffset CreatedDate { get; set; }
        public string UpdatedBy { get; set; } = string.Empty;
        public DateTimeOffset UpdatedDate { get; set; }
    }
}
